central/networkbaseline/service/service_impl.go

```diff
@@ -161,11 +162,24 @@ func (s *serviceImpl) getBaselinePeerByEntityID(
 
 	peers := baseline.GetPeers()
 	for _, peer := range peers {
+		peerType := peer.GetEntity().GetInfo().GetType()
+		peerId := peer.GetEntity().GetInfo().GetId()
 		key := networkgraph.Entity{
-			Type: peer.GetEntity().GetInfo().GetType(),
-			ID:   peer.GetEntity().GetInfo().GetId(),
+			Type: peerType,
+			ID:   peerId,
 		}
 		result[key] = peer
+		// Insert second peer entry for the masked deployment ID
+		// that can be issued by the network graph deployment masker
+		if peerType == storage.NetworkEntityInfo_DEPLOYMENT {
+			deploymentName := peer.GetEntity().GetInfo().GetDeployment().GetName()
+			maskedID := uuid.NewV5FromNonUUIDs(peerId, deploymentName).String()
+			maskedKey := networkgraph.Entity{
+				Type: peerType,
+				ID:   maskedID,
+			}
+			result[maskedKey] = peer
```

Comment:
So we are adding it twice to the `result` map?

I am lacking context of what this map precisely does, so I would appreciate some explanation to get more context.